.github/workflows/example-publish.yml

```diff
@@ -37,4 +37,4 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: build-provenance
```

Comment:
should we remove this name field from both?

Reply:
Not sure. We could so both the artifact and its provenance would be in the same archive. I'd be fine with either but regardless, it should be in a follow-on change.